Domain/viewModelInterface.go

```diff
@@ -1,4 +1,17 @@
 package Domain
 
-type HelloWorldViewModel interface {
+type UserViewModelInterface interface {
```

Comment:
Also Client ?
P.S. Is it a good idea to call "Client" all Bank clients?
Cause of user - could be any user, i.e. not a client.